src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -381,7 +381,13 @@ export default React.memo(function GamePage(): JSX.Element | null {
                     <DotsMenu gameInfo={gameInfo} handleUpdate={handleUpdate} />
                   </div>
                   <div className="infoWrapper">
-                    <Genres genres={wikiInfo?.pcgamingwiki?.genres || []} />
+                    <Genres
+                      genres={
+                        gameInfo.extra?.genres ||
```

Comment:
could this be an empty array? because in that case it `[] || wikiInfo...` would still be `[]` even if we have genres from pc gaming wiki and for some reason the gog or amazon info has no genres

Reply:
good point

Reply:
As far as I can see the array is empty for some GOG titles, that aren't games and in case of `Galaxy Integration Demo`. Amazon seems to always have genres. I'd say we are safe